docs/building-with-jugalbandi/technical-guide/tutorials/quickstart.md

```diff
@@ -72,9 +73,15 @@ $ python3 -m http.server 5000
 
 6. Start a new shell session and start loophole tunnel for port 5000 on your system
 
-```bash
-$ ./loophole http 5000
-```
+   - For **Loophole**, use the following command:
+     ```bash
+     $ ./loophole http 5000
+     ```
+
+   - For **ngrok**, use the following command:
+     ```bash
+     $ ngrok http 5000
+     ```
 
 Copy the tunnel url from the loophole shell and add it to `PUBLIC_URL_PREFIX` in `.env-dev` file. ![](../../../assets/quickstart-loophole-5000.png)
```

Comment:
We can remove loophole and just keep the shell or we can say something like (copy the tunnel url from the example loophole shell shown below)